src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -1089,6 +1089,68 @@ internal void SelectAllCategories(object parameter)
             }
         }
 
+        /// <summary>
```

Comment:
@ramramps , Do you mean to start from the deepest layer, i.e.from the last subcategory? But what is the reason? In any case we have to set false for all of them and order doesn't matter. Or it does?